app/kuma-dp/pkg/dataplane/accesslogs/server.go

```diff
@@ -33,10 +33,21 @@ func (s *accessLogServer) NeedLeaderElection() bool {
 }
 
 func NewAccessLogServer(dataplane kumadp.Dataplane) *accessLogServer {
+	// Log an explicit error when access log address is too long
+	// see: https://man7.org/linux/man-pages/man7/unix.7.html
+	var address = fmt.Sprintf("/tmp/kal-%s-%s.sock", dataplane.Name, dataplane.Mesh)
+	if len(address) > 108 {
+		logger.Error(nil, fmt.Sprintf(`The name of your dataplane is too long: %s. 
+										It is important that the address to store your access logs doesn't have more than 108 characters. 
+										The actuall address is: %s.
+										Please reduce the size of your dataplane name.`,
+			dataplane.Name,
+			address))
+	}
```

Comment:
Can we do something more automatic here?
We are not so attached to this name, to be honest. We can just as well either:
1) trim the name to 108 chars
2) use UUID

As long as we log this path to the file, it's fine.

Reply:
I thought about using a UUID but I thought it would complicate debug. 
In which file do you want to log the path ? In the normal logs (using `logger.Info(...)` ) ?